settlement/base.go

```diff
@@ -159,14 +143,13 @@ func (b *BaseLayerClient) fetchSequencersList() ([]*types.Sequencer, error) {
 	return sequencers, nil
 }
 
-func (b *BaseLayerClient) validateBatch(batch *types.Batch) error {
+func (b *BaseLayerClient) validateBatch(batch *types.Batch) {
```

Comment:
Not sure why to remove the `error`
I don't think it's good idea to have panic scattered around

The internal functions should return errors, to be aggregated and panic upon if needed

Reply:
makes sense